docs/wiki/[Variables]-configure_connectivity_resources.md

```diff
@@ -535,7 +662,117 @@ List of [Azure] resource IDs used to identify spoke Virtual Networks associated
 
 ### Configure hub networks (Virtual WAN)
 
-_Not implemented yet, coming soon._
+Define zero or more VWAN hub networks as a list of objects, each containing configuration values covering an address space, DDOS protection plan, expressroute gateway, vpn gateway and Azure firewall.
```

Comment:
DDoS Protection plan is setup outside of the VWAN, and we actually don't have a means by which to link it to Virtual Hubs currently.

Reply:
Cool - will change